src/usb_cam.cpp

```diff
@@ -26,48 +26,48 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
+#define CLEAR(x) memset(&(x), 0, sizeof(x))
```

Comment:
Could we consider a template instead?

Reply:
@twdragon yes for sure! This was just the existing code as before but now after the refactor we can improve it